src/components/container/FeaturedJobs/FeaturedJobs.js

```diff
@@ -0,0 +1,34 @@
+import { staticFeaturedJobs } from '../../../state/models/staticFeaturedJobs';
+import viewMoreIcon from './assets/icons/ViewMore.svg';
```

Comment:
Please use camelCase over PascalCase for asset file names:

e.g. `viewMore.svg` rather than `ViewMore.svg`